include/boost/process/v2/detail/impl/exe.ipp

```diff
@@ -0,0 +1,71 @@
+// Copyright (c) 2022 Klemens D. Morgenstern
+//
+// Distributed under the Boost Software License, Version 1.0. (See accompanying
+// file LICENSE_1_0.txt or copy at http://www.boost.org/LICENSE_1_0.txt)
+#ifndef BOOST_PROCESS_V2_IMPL_DETAIL_EXE_IPP
+#define BOOST_PROCESS_V2_IMPL_DETAIL_EXE_IPP
+
+#include <boost/process/v2/detail/exe.hpp>
+#include <boost/process/v2/detail/config.hpp>
+#include <boost/process/v2/detail/last_error.hpp>
+#include <boost/process/v2/detail/throw_error.hpp>
+
+BOOST_PROCESS_V2_BEGIN_NAMESPACE
+
+namespace detail
+{
+
+namespace ext
+{
+
+#if defined(_WIN32)
+
+HANDLE open_process_with_debug_privilege(pid_type pid, error_code & ec) {
```

Comment:
```suggestion
HANDLE open_process_with_debug_privilege(pid_type pid, error_code & ec) 
{
```

Personal preference.

Reply:
I'd also like to get a few lines on what this does and why we need it. Not to mention that `basic_process`.

Reply:
Not to mention that basic_process? What do you mean by that? I added a few lines giving explanation comments.

Reply:
I probably should've finished the sentence - the basic_process you get as result of launching already has the debug privilege I think.

Reply:
well that's only if you launched it yourself it sounds like you meant, however this can access any process even ones you didn't run yourself. It allows you to read information from processes run by other users as well as admin run processes.